lib/controller.dart

```diff
@@ -197,6 +229,9 @@ bool _interactionMatchesOperation(
 }
 
 void _computeChartBuckets(List<model.Chart> charts) {
+  _filterValuesCount = 0;
+  _comparisonFilterValuesCount = 0;
```

Comment:
Instead of adding these counts as extra state, would you be able to compute them were needed by summing the chart field buckets?

Reply:
I have been calculating this wrong, it needs to be super set of the field buckets and need to be counted at the global filter level and not at the bucket level. Fixed.